proto/src/lib.rs

```diff
@@ -266,3 +266,6 @@ pub mod tendermint {
         include!("gen/tendermint.p2p.rs");
     }
 }
+
+#[cfg(feature = "rpc")]
+pub const FILE_DESCRIPTOR_SET: &[u8] = include_bytes!("gen/proto_descriptor.bin");
```

Comment:
FYI, I ran `cargo check-all-features --release` (#2210) locally, and it failed on this branch. It's also failing on main, but it was passing earlier this week, so will dig around a bit more and see if we can unbreak it. Doesn't need to block here.